test/packages/benchmarks/rally_benchmark/_dev/benchmark/rally/logs-benchmark.yml

```
@@ -0,0 +1,17 @@
+---
+description: Benchmark 20000 events ingested
+input: filestream
+vars: ~
+data_stream:
+  name: testds
+warmup_time_period: 10s
+corpora:
+  generator:
+    total_events: 20000
+    template:
+      type: gotext
+      path: ./logs-benchmark/template.ndjson
+    config:
+      path: ./logs-benchmark/config.yml
+    fields:
+      path: ./logs-benchmark/fields.yml
```

It looks like all the files are known and the only variable is the dir, should we require only the directory name?
```suggestion
    config:
      template_type: gotext
      dir: ./logs-benchmark
```

Or we want to support to reuse files between different benchmarks?

---

this is still an open discussion: the spec for system scenario was defined this way. also in the spec we can either have a path to a file or raw inline content. I'd keep it this way, even considering changing it will be another one breaking change :)

---

Oh ok, the raw and the possibility of reusing files can be good reasons to keep it like this.